.travis.yml

```diff
@@ -1,14 +1,18 @@
 language: go
 
 go:
-  - 1.2
-  - 1.3
+  - 1.6
+  - 1.7
+  - 1.8
   - tip
 
 before_install:
```

Comment:
Yes, this should be possible, but I am not sure if with the core Golang testing. That's why I also added [gocheck](http://labix.org/gocheck) at some point.